ffi/SDL2_0.lua

```diff
@@ -170,6 +181,19 @@ function S.waitForEvent(usecs)
             genEmuEvent(ffi.C.EV_SYN, ffi.C.SYN_REPORT, 0)
         elseif event.type == SDL.SDL_MULTIGESTURE then -- luacheck: ignore 542
             -- TODO: multi-touch support
+        elseif event.type == SDL.SDL_WINDOWEVENT
+            and (event.window.event == SDL.SDL_WINDOWEVENT_RESIZED
+                 or event.window.event == SDL.SDL_WINDOWEVENT_SIZE_CHANGED) then
+            local x = 0
```

Comment:
@poire-z I should probably call this w and h, right?

Reply:
yep

Reply:
btw, why w=0 and h=1 (and not both 0 or 1) ?

Reply:
For front I pretend it's some kind of custom kernel event.

```lua
-- type (MSC), property (w), value (new_size_w)
genEmuEvent(ffi.C.EV_MSC, w, new_size_w)
```

Would you rather I try passing a non-standard value (like a serialized Lua string) so it can be done as a single event instead of three separate events?

Reply:
No no, no pb with that.
Just curious why `w=0, h=1` . And why not the more symetrical `w=1, h=1`   or   not `w=0, h=0`

Reply:
oh, ok, these are just codes, 0 means width, 1 means height - and not actual width and height !